tools/summaries/fetch_ranking_based_place_summaries.py

```diff
@@ -14,6 +14,9 @@
 """Generate ranking-based summaries for all child places of a specific place
 type in a parent place.
 
+This was the script used to generate the summaries used in mid-January 2024,
```

Comment:
curious, is it just the summaries that are no longer used or is this script also no longer used?

Reply:
Both, the summaries are no longer used, and the current script isn't used for the current version of summaries that are live on our site now.

Reply:
can you add a note that this script is no longer used

Reply:
Done!